### PR TITLE
Fix references to types when registering secrets managers

### PR DESCRIPTION
**Pull request by strickvl**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: docs/book/advanced-guide/manage-your-secrets.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ register your secrets manager in the following way:
 
 ```shell
 zenml integration install aws
-zenml secrets-manager register AWS_SECRETS_MANAGER_NAME --flavor=aws
+zenml secrets-manager register AWS_SECRETS_MANAGER_NAME --flavor aws
```
Comment (AlexejPenner) [inline docs/book/advanced-guide/manage-your-secrets.md:216]: Is it really wrong to use = with --flavor ?

Comment (strickvl) [reply docs/book/advanced-guide/manage-your-secrets.md:216]: No not wrong, but nowhere else really do we use the `=` for flavor assignment. It was just a smaller thing I noticed while I was in the document. The main change is the other one.

Comment (strickvl) [reply docs/book/advanced-guide/manage-your-secrets.md]: I reverted this specific change now, @AlexejPenner, following your comment.

**Review by fa9r — APPROVED** [review]
LGTM

**Pull request merged**